conf/base/catalog.yml

```diff
@@ -24,4 +24,76 @@ oa_int_downstream_incoming_citations:
   path: "s3://alphafold-impact/data/02_intermediate/openalex/works/cited_by/downstream/${globals:get.work_id, undefined}/"
   credentials: s3_credentials
 
+# CRUNCHBASE
+# Raw
+_cb: &_cb
+  type: pandas.CSVDataset
+  credentials: nesta_s3_credentials
 
+cb_acquisitions_raw:
```

Comment:
A general comment for consistency: I've used dot notation to represent the hierarchy of the structure, as it helps keep short sentences separate from any YAML hierarchy (ie. `data_collection` vs. `gtr.raw`). I also do so because for OmegaConfig it is convention to use dot notation for naming keys representing nested configurations (this also helps with OC's resolvers).

Reply:
Btw, in case it is not clear:

- **YAML Structure**: Uses indentation and nesting to represent hierarchy.
  ```
  parent:
    child:
      property: value
  ```
- **Dot Notation in OmegaConfig**: Used in key names to mimic a hierarchical structure in a flat key-value pair.
  ```
  parent.child.property: value
  ```

Reply:
Thanks! The last few entries [here](https://github.com/innovation-growth-lab/alphafold-impact/blob/8aaff82510c623684edaec3c515b5e953dd92f70/conf/base/catalog.yml#L73) seem to use both styles. Are the last two following a legacy format and need updating or is there something I'm misunderstanding about the notation?

Reply:
The last two entries are using the legacy format. They have been updated in [this PR](https://github.com/innovation-growth-lab/alphafold-impact/pull/28)